File: Curso 1/Programación/Ejercicios/Ejercicios Java #1/Ejercicio1.java
/* Título: Ejercicio 1
* Algoritmo: Mostrar el cuadrado de un número
* Fecha: 13/11/2022
* Autor: Hugo Pelayo
* */

import java.io.*;

public class Ejercicio1 {
    public static void main(String[] args) throws IOException {
        InputStreamReader inputReader = new InputStreamReader(System.in);
        BufferedReader reader = new BufferedReader(inputReader);
        float numero;

        do {

            System.out.print("Introduce un número real por favor: ");
            numero = Float.parseFloat(reader.readLine());

            if (numero > 0)
                System.out.printf("[ %.3f ] al cuadrado es [ %.3f ]\n", (numero), (float)(Math.pow(numero, 2)));
        }
        while (numero >= 0);
    }
}

File: Curso 1/Programación/Ejercicios/Ejercicios Python/Ejercicios Python #6/Ejercicio1.py
# Ejercicio 1
# Algoritmo: Factorial de un número entero
# Fecha: 12.12.2022
# Auor: Hugo Pelayo

def factorial(num):
    return 1 if num <= 0 else num * factorial(num - 1)

numero = int(input('Introduce un número entero: '))
print(f'El factorial de {numero} es: {factorial(numero)}')